packages/blockchain-wallet-v4-frontend/src/modals/SimpleBuy/EnterAmount/Checkout/template.success.tsx

```diff
@@ -456,21 +456,19 @@ const Success: React.FC<InjectedFormProps<{}, Props> & Props> = props => {
             </Amounts>
           )}
 
-        {(!props.isSddFlow || props.orderType === 'SELL') &&
-          props.pair &&
-          min > max && (
-            <Amounts>
-              <CustomErrorCartridge
-                role='button'
-                data-e2e='sbEnterAmountNotEnoughFundsForSell'
-              >
-                <FormattedMessage
-                  id='modals.simplebuy.checkout.not_enough_funds_for_sell'
-                  defaultMessage='Not Enough funds for Sell'
-                />
-              </CustomErrorCartridge>
-            </Amounts>
-          )}
+        {props.orderType === 'SELL' && props.pair && min > max && (
+          <Amounts>
+            <CustomErrorCartridge
+              role='button'
+              data-e2e='sbEnterAmountNotEnoughFundsForSell'
+            >
+              <FormattedMessage
+                id='modals.simplebuy.checkout.not_enough_funds_for_sell'
+                defaultMessage='Not Enough funds for Sell'
+              />
+            </CustomErrorCartridge>
+          </Amounts>
+        )}
```

Comment:
this has to be part only of SELL flow

Reply:
No idea what's this about. But `min > max`? You sure?

Reply:
yes this is corner case where we have amount(max) lower than min